frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -169,16 +176,36 @@ export const getServerlessData = (resource: K8sResourceKind) => {
       maxpods: '',
       concurrencytarget: '',
       concurrencylimit: '',
+      autoscale: {
+        autoscalewindow: '',
+        autoscalewindowUnit: '',
+        defaultAutoscalewindowUnit: '',
+      },
+      concurrencyutilization: '',
     },
   };
+  const windowRegEx = /^[0-9]*|[a-zA-Z]*/g;
   if (getResourcesType(resource) === Resources.KnativeService) {
-    const annotations = _.get(resource, 'spec.template.metadata.annotations');
+    const {
+      spec: {
+        template: { metadata, spec },
+      },
+    } = resource;
+    const annotations = metadata?.annotations;
+    const autoscalewindowAnnotaion = annotations?.[KNATIVE_AUTOSCALEWINDOW_ANNOTATION] || '';
+    const autoscalewindow = autoscalewindowAnnotaion.match(windowRegEx);
     serverlessData = {
       scaling: {
-        minpods: _.get(annotations, 'autoscaling.knative.dev/minScale', 0),
-        maxpods: _.get(annotations, 'autoscaling.knative.dev/maxScale', ''),
-        concurrencytarget: _.get(annotations, 'autoscaling.knative.dev/target', ''),
-        concurrencylimit: _.get(resource, 'spec.template.spec.containerConcurrency', ''),
+        minpods: annotations?.[KNATIVE_MINSCALE_ANNOTATION] || 0,
```

Comment:
I know it's been there but looks odd as we setting in explicitly on client side. Maybe we can leave it empty ad other fields are unless we are fetching defaults and not add annotations while creation if empty and let backend handle it, WDYT??